pages/portfolio-style.css

```diff
@@ -0,0 +1,125 @@
+
+.container {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr;
+    background-color: rgb(252, 246, 229);
+    height: 100vh;
+    width:100vw;
+    position: relative;
+    /* border: solid red; */
+}
```

Comment:
Consider re-using CSS where you can instead of duplicating it in different files.